code/WorkInProgress/MechanicMadness.dm

```diff
@@ -1529,13 +1537,13 @@
 	//Called when mechanics_holder tries to fire out signals
 	proc/runFilter(var/comsig_target, atom/receiver, var/signal)
 		if(!(receiver in src.outgoing_filters))
-			return 0 //Not filtering this output, let anything pass
+			return src.single_output? 2 : 0 //Not filtering this output, let anything pass
```

Comment:
Can we get some kind of `#define` for what these different return values mean?

Reply:
Good idea. Where would be the best place to put the defines?

Reply:
I guess a new file in std/defines?